src/fu-config.c

```diff
@@ -247,6 +249,14 @@ fu_config_reload(FuConfig *self, GError **error)
 		self->show_device_private = TRUE;
 	}
 
+	/* whether to allow emulation to work */
+	self->allow_emulation =
+	    g_key_file_get_boolean(keyfile, "fwupd", "AllowEmulation", &error_allow_emulation);
+	if (!self->allow_emulation && error_allow_emulation != NULL) {
+		g_debug("failed to read AllowEmulation key: %s", error_allow_emulation->message);
+		self->allow_emulation = TRUE;
```

Comment:
I think we should default this to false too

Reply:
Fixed.